### DISCUSSION TITLE
[6차 스프린트] 6차 스프린트 목표 및 할 일 정리

### DISCUSSION THREAD

**Discussion by Minjoo522** [woowacourse-teams/2024-devel-up#580] [백로그 논의]
#### 논의가 필요한 내용

이제 6차 스프린트 시작!
6차 스프린트 기간 동안 각자 생각하는 팀의 방향성 + 목표 / 개인적인 목표 + 중심으로 하고싶은 것을 적어주면 좋을 것 같아요.
편의를 위해서 아래처럼 작성해야 할 템플릿을 정의해 둘게요! (💡 더 필요한 게 있다고 생각하면 언제든지 말해주세요)

- **기한 : 10월 2일(수) 회의 전까지**

---

# 템플릿
## 6차 스프린트 내가 생각하는 팀의 방향성
- 내용

## 6차 스프린트 내가 생각하는 팀의 목표(해야할 일)
- 내용

## 6차 스프린트 개인적인 목표
- 내용

## 6차 스프린트 개인 비중
|프로젝트|미션|취준|기타|
|:--:|:--:|:--:|:--:|
|00%|00%|00%|내용(00%)|

## 기타 적고 싶은 것

**Comment by Parkhanyoung** [comment]
### 6차 스프린트 내가 생각하는 팀의 방향성

요구사항을 잘 챙기자

### 6차 스프린트 내가 생각하는 팀의 목표(해야할 일)

요구사항 충족 및 팀원들 각각의 니즈가 충족될 수 있는 환경 조성

### 6차 스프린트 개인적인 목표

새로운 기능을 무조건 많이 쳐내는 것보다 기존의 작업들을 더 공고히 하는 방향이 되면 어떨까 싶습니다!
예를 들면, 이전에 빠졌던 삭제/수정을 추가한다던지, FE 요구사항인 접근성을 적용한다던지, storybook을 작성하는 등의 업무가 있을 것 같습니다!


### 6차 스프린트 개인 비중
| 프로젝트 | 미션 | 취준 | 기타 |
|--------|--------|--------|--------|
| 30% | 30% | 20% | 개인 학습(20%) |

**Comment by le2sky** [comment]
### 6차 스프린트

### 6차 스프린트 내가 생각하는 팀의 목표(해야할 일)

사용자 유치 시도

### 6차 스프린트 개인적인 목표

몰입

### 6차 스프린트 개인 비중
| 프로젝트 | 미션 |  기타 |
|--------|--------|--------|
| 50% | 10% | 40% |

**Comment by alstn113** [comment]
### 6차 스프린트 내가 생각하는 팀의 방향성

요구 사항을 잘 구현

### 6차 스프린트 내가 생각하는 팀의 목표(해야할 일)

성능 테스트 공부

### 6차 스프린트 개인적인 목표

성능 테스트 부분 잘 배우기

### 6차 스프린트 개인 비중
| 프로젝트 | 미션 | 취준 | 기타 |
|--------|--------|--------|--------|
| 40% | 20% | 00% | 개인공부(40%) |

**Comment by chosim-dvlpr** [comment]
### 6차 스프린트 내가 생각하는 팀의 방향성
더 견고하게 만들기

### 6차 스프린트 내가 생각하는 팀의 목표(해야할 일)
지금 있는 기능까지 완료하고, (좋아요, 스크랩, 수정 삭제까지는 추가해도 상관없음)
기존의 것을 더 견고하게 만드는 게 중요하다고 생각해요!
가벼운 기능 여러 개 추가하는 것보다 기존에 있는 것을 충분히 활용해서 깊이 파는 것이 중요하다는 생각이 들어요
+사용자 유치 후 유지보수하는 과정이 필요할 것 같아요

### 6차 스프린트 개인적인 목표
새로운 기능 추가도 나쁘지 않지만, 기능 추가의 비중은 많이 두기보다 기존의 것을 리팩토링하고 딥하게 개선하는 작업을 하고 싶어요!
개인적으로 SSR (서버사이드 렌더링)에 관심이 많아서 시간과 여유가 된다면 .... 메인 페이지 정도만 SSR을 적용해보면 좋겠다는 생각도 있어용

### 6차 스프린트 개인 비중
| 프로젝트 | 미션 | 취준 | 기타 |
|--------|--------|--------|--------|
| 30% | 30% | 30% | 개인 공부 (10%) |

**Comment by brgndyy** [comment]
### 6차 스프린트

### 6차 스프린트 내가 생각하는 팀의 목표(해야할 일)

공통 요구 사항을 챙기고, 실제 사용자에게 홍보를 한다.

### 6차 스프린트 개인적인 목표

6차 기획 회의에서 나오는 요구사항과 더불어 프론트 단에서의 미션 (접근성) 부분도 견고히 하고 싶습니다! 

### 6차 스프린트 개인 비중
| 프로젝트 | 미션 | 취준 | 기타 |
|--------|--------|--------|--------|
| 30% | 20% | 30% | 개인 학습(20%) |

**Comment by robinjoon** [comment]
### 6차 스프린트

### 6차 스프린트 내가 생각하는 팀의 목표(해야할 일)

인기 풀이, 디스커션 기능 추가 / 요구사항 만족 / 사용자 유치

### 6차 스프린트 개인적인 목표
#### 프로젝트
1. 합리적인 논증에 따른 목표 성능 설정
2. 이를 달성하기 위한 여러 방법 나열
3. 실험 및 검증
4. 더욱 잘 설계된 미션 제공을 통한 서비스 유지 가능성 높이기
#### 미션
1. 요구사항 만족하기
2. 모래주머니(컨벤션, 클린 코드 등) 풀지 않기
#### 알고리즘
1. 그동안 풀었던 문제들 다 다시 풀어보기
#### 취준
1. 작은 기업이던 큰 기업이던 상관하지 말고 일단 이력서 넣기
#### CS
1. CS 면접 질문 사례들 주에 4개 이상 답 적어보기
### 6차 스프린트 개인 비중
| 프로젝트 | 미션 | 알고리즘 | CS | 취준|
|--------|--------|--------|--------|--------|
| 30% | 10% | 30% | 20% |10%|

**Comment by Minjoo522** [comment]
## 6차 스프린트 내가 생각하는 팀의 방향성
- 요구 사항 먼저 잘 완성하기

## 6차 스프린트 내가 생각하는 팀의 목표(해야할 일)
- 요구 사항 완성
- MVP 구현 완료
- 사용자 유치

## 6차 스프린트 개인적인 목표
- 뭐 하나라도 몰입해서 하고싶다.

## 6차 스프린트 개인 비중
|프로젝트|미션|취준|기타|
|:--:|:--:|:--:|:--:|
|30%|30%|30%|private(10%)|

**Comment by lilychoibb** [comment]
## 6차 스프린트 내가 생각하는 팀의 목표(해야할 일)
- 요구 사항 충족

## 6차 스프린트 개인적인 목표
- 요구 사항 충족
- 기존 코드 또는 설계 개선

## 6차 스프린트 개인 비중
|프로젝트|미션|취준|
|:--:|:--:|:--:|
|30%|20%|50%|
